recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
-{% set version = '1.1.0' %}
-{% set sha256 = '3a850036bf42053c74bfc52c063323ca78e40ba1f326b01777da5750a143631a' %}
+{% set version = "1.1.1" %}
+{% set sha256 = '8ec8e9087d13c99b354ab6d8b4cadb1758633db5946ff95a6bc7ac538b6d7b3d' %}
```

Comment:
Intriguing, it changes the string quotes in version and leaves them alone in the checksum. Any idea why this is @CJ-Wright?

Reply:
I'm not certain. I think we started normalizing some of the variable definitions, but maybe not all? We can do the rest though, it should be an easy enough fix. Ideally this would all be done in some nested loops so the formatting world be uniform across feedstocks.

Reply:
My temptation would be to skip normalizing strings. Is there a reason that should be desired?

Reply:
Hmm ok, we should do it for the spaces though, that way we PR fixes before the new liner comes out.